velox/functions/prestosql/MapKeyExists.h

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
I think we can have a MapFunctions.h to contain all simple map functions (similar to ArrayFunctions.h), instead of having one file for each function. You can just rename this file name to MapFunctions.h.